pages/1.13/cli/command-reference/dcos-license/dcos-license-get/index.md

```diff
@@ -2,43 +2,64 @@
 layout: layout.pug
 navigationTitle:  dcos license get
 title: dcos license get
-menuWeight: 1
+menuWeight: 2
 excerpt: Displaying the cluster licenses
-
 enterprise: true
 ---
 
 # Description
-The `dcos license get` command allows you to view the cluster licenses.
+The `dcos license get` command retrieves a specific license associated with this cluster. When no LICENSE_ID is passed, the current license is returned. This command outputs to `stdout` by default.
+
 
 # Usage
 
 ```bash
-dcos license get [<id>|active] [--output <file_path>] [--decryption-key]
-```
+ dcos license get [OPTIONS] [LICENSE_ID]
+ ```
 
 # Options
 
-| Name, shorthand |  Description |
-|---------|-------------|
-|   |   |
-| `--output`   |   Store the license in a file. |
+| Name |  Description |
+|-------------------|-------------------|
+| `--help`   |    Show this message and exit. |
 | `--decryption-key`   |  Get the key to decrypt license audit records. |
 
+## Positional Arguments
 
-# Positional arguments
+| Name |  Description |
+|-------------------|-------------------|
+| `LICENSE_ID` | ID of license to be retrieved. If no `LICENSE_ID` parameter is passed, the current license will be retrieved. |
 
-| Name, shorthand |  Description |
-|---------|-------------|
-| `<id> `   |   The ID of the license. |
-| `active`   |   The active license. |
-| `<file_path>`    |  The path to a file to store the license |
+# Examples
+
+```json
+dcos license get
```

Comment:
side note @ArmandGrillet and @masonse  , do you need to be an admin to get this information?